includes/EmailNotification.php

```diff
@@ -544,8 +544,8 @@ private function compose( \User $user ) {
 				'exception' => new \Exception(),
 			];
 
-			if ( \F::app()->wg->DisableOldStyleEmail ) {
-				$emailContext['issue'] = 'SOC-2290';
+			if ( !F::app()->wg->Title->inNamespace(NS_SPECIAL) || $this->action !== "overwrite" ) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Refactor this code to rely on RequestContext instead of context-dependent globals. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3AContextualWgVariableUsage)